apps/prairielearn/src/sync/fromDisk/assessments.ts

```diff
@@ -133,20 +138,28 @@ function getParamsForAssessment(
             maxAutoPoints: alternative.maxAutoPoints ?? question.maxAutoPoints ?? null,
             autoPoints: alternative.autoPoints ?? question.autoPoints ?? null,
             manualPoints: alternative.manualPoints ?? question.manualPoints ?? null,
-            forceMaxPoints: _.has(alternative, 'forceMaxPoints')
-              ? alternative.forceMaxPoints
-              : _.has(question, 'forceMaxPoints')
-                ? question.forceMaxPoints
-                : false,
-            triesPerVariant: _.has(alternative, 'triesPerVariant')
-              ? alternative.triesPerVariant
-              : _.has(question, 'triesPerVariant')
-                ? question.triesPerVariant
-                : 1,
-            advanceScorePerc: alternative.advanceScorePerc,
-            gradeRateMinutes: _.has(alternative, 'gradeRateMinutes')
-              ? alternative.gradeRateMinutes
-              : questionGradeRateMinutes,
+            forceMaxPoints:
+              Object.prototype.hasOwnProperty.call(alternative, 'forceMaxPoints') &&
+              alternative.forceMaxPoints !== undefined
+                ? alternative.forceMaxPoints
+                : Object.prototype.hasOwnProperty.call(question, 'forceMaxPoints') &&
+                    question.forceMaxPoints !== undefined
+                  ? question.forceMaxPoints
+                  : false,
+            triesPerVariant:
+              Object.prototype.hasOwnProperty.call(alternative, 'triesPerVariant') &&
+              alternative.triesPerVariant !== undefined
+                ? alternative.triesPerVariant
+                : Object.prototype.hasOwnProperty.call(question, 'triesPerVariant') &&
+                    question.triesPerVariant !== undefined
+                  ? question.triesPerVariant
+                  : 1,
+            advanceScorePerc: alternative.advanceScorePerc ?? 0,
```

Comment:
added a fallback of 0 here.

Reply:
Fallback corrected in 854cc5fc2.

Reply:
@nwalters512 It looks like the previous code fell back to `0` instead of undefined. is that going to cause any issues?